go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/Microsoft/hcsshim v0.8.10 // indirect
 	github.com/containerd/continuity v0.0.0-20201119173150-04c754faca46 // indirect
 	github.com/dustin/go-humanize v1.0.0
+	github.com/emicklei/go-restful v2.15.0+incompatible
```

Comment:
There must be a import with the old go-restful somewhere lingering around, this must be converted to use v3 as well

Reply:
or go mod tidy is missing

Reply:
You're right, go mod tidy did the trick